apps/test/integration/levelSolutions/applab2/ec_setprop.js

```diff
@@ -220,7 +235,7 @@ module.exports = {
       editCode: true,
       levelHtml:
         '<div xmlns="http://www.w3.org/1999/xhtml" id="designModeViz" class="appModern withCrosshair" style="width: 320px; height: 450px; display: none;"><div class="screen" tabindex="1" id="screen1" style="display: block; height: 450px; width: 320px; left: 0px; top: 0px; position: absolute; z-index: 0;"><input type="range" value="50" min="0" max="100" step="1" id="my_slider" style="margin: 0px; padding: 0px; width: 150px; height: 24px; position: absolute; left: 75px; top: 95px;" /></div></div>',
```

Comment:
This test was failing on Chrome because for an `input type="range"` element when you retrieve the value, Chrome will coerce the value to one that conforms to the min/max/step constraints set on the range control.

In this particular case, the control is given min=1, max=101, step=3. Its value is given as 51, which does not conform to the constraint:

```
(value) % (step) - (min) == 0
   51   %    3   -   1   == -1 // Nonconforming
```

With a little manual testing I discovered that Chrome and Firefox both enforce the constraint this way:

Chrome:
![Screenshot from 2020-04-29 10-31-50](https://user-images.githubusercontent.com/1615761/80630185-af154680-8a08-11ea-9e56-748bb14a571f.png)

Firefox:
![Screenshot from 2020-04-29 10-39-16](https://user-images.githubusercontent.com/1615761/80630191-b177a080-8a08-11ea-8439-80e3a769011e.png)


But Phantom must not for this to be passing on Drone right now.  Eventually I'd love to add a test for this value coercion, but since that's not currently covered by this test I've just changed the value to 52 which should still pass on Phantom but also pass on Chrome since it does conform.